v3/setup/images-control.sh

```diff
@@ -6,11 +6,11 @@ if [ "${NODE_ROLE}" != "control" ]; then
     exit 0
 fi
 
-if [ "$(etcdctl get images-control-bootstrapped)" == "true" ]; then
+if [ "$(etcdctl get /bootstrap.service/images-control-bootstrapped)" == "true" ]; then
```

Comment:
at that point, I guess I could move the `/images` dir into `bootstrap.service`. But having to type out `bootstrap.service/images/control/flight-director` is a lot. I don't know